config/packages/pimcore_admin.yaml

```diff
@@ -0,0 +1,7 @@
+# see https://pimcore.com/docs/pimcore/current/Development_Documentation/Best_Practice/Security_Concept.html#page_Content-Security-Policy
+#   and https://content-security-policy.com/#directive for more information
+# IMPORTANT: Please note enabling CSP headers for admin interface will make DataObject WYSIWYG editor menu unresponsive
+#   as CKeditor 4 does not completely support it.
```

Comment:
Does it make sense to basically "destroy" the WYSIWYG feature within DataObjects with that setting? I guess we are using that field type pretty frequently. 🤔

Reply:
@aarongerig It looks like this bug has been fixed by CKEditor. instanceIhttps://github.com/ckeditor/ckeditor4/commit/3ee603539f24cbebf0c0c24ec1b3eb48d678570b

Reply:
@AndiKeiser Cool. Pimcore is using CKEditor v4.18.0 and since this fix was introduced with v4.7.0 we are good.
@alexloetscher95 Could you remove that comment then?